deploy/olm-catalog/awx-operator/manifests/awx-operator.clusterserviceversion.yaml

```diff
@@ -127,6 +128,23 @@ spec:
         - urn:alm:descriptor:com.tectonic.ui:advanced
         - urn:alm:descriptor:io.kubernetes:Secret
         - urn:alm:descriptor:com.tectonic.ui:fieldDependency:tower_ingress_type:Ingress
+      - displayName: Tower LoadBalancer Annotations
+        path: tower_loadbalancer_annotations
+        x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:advanced
+        - urn:alm:descriptor:com.tectonic.ui:text
+        - urn:alm:descriptor:com.tectonic.ui:fieldDependency:tower_ingress_type:LoadBalancer
+      - displayName: Tower LoadBalancer Protocol
+        path: tower_loadbalancer_protocol
+        x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:advanced
+        - urn:alm:descriptor:com.tectonic.ui:select:http
+        - urn:alm:descriptor:com.tectonic.ui:select:https
+      - displayName: Tower LoadBalancer Port
+        path: tower_loadbalancer_port
+        x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:advanced
+        - urn:alm:descriptor:com.tectonic.ui:number
```

Comment:
Need to add `urn:alm:descriptor:com.tectonic.ui:fieldDependency:tower_ingress_type:LoadBalancer`